lambdas/drivingpermitcheck/src/test/java/uk/gov/di/ipv/cri/drivingpermit/api/handler/DrivingPermitHandlerTest.java

```diff
@@ -67,7 +68,7 @@ class DrivingPermitHandlerTest {
 
     @Mock private ServiceFactory mockServiceFactory;
     @Mock ThirdPartyAPIServiceFactory mockThirdPartyAPIServiceFactory;
-    @Mock ThirdPartyAPIService mockDcsThirdPartyDocumentGateway;
+    @Mock ThirdPartyAPIService mockThirdPartyDocumentGateway;
```

Comment:
Can we mock the implementations rather than the interface. Then we can add a line to the test like
verify(mockIdentityVerificationService).verifyIdentity(any(DrivingPermitFormData.class), mockDcsThirdPartyDocumentGateway) or verify(mockIdentityVerificationService).verifyIdentity(any(DrivingPermitFormData.class), mockDvaThirdPartyDocumentGateway)

Reply:
Corrected.